### ISSUE TITLE
reduce number of menu items

### ISSUE THREAD

**Issue by bob-carpenter** [stan-dev/stan-dev.github.io#27]
This is from @andrewgelman, who wants to reduce the number of menu items on the top page in order to focus user search.  The plan will require more clicks to get to some things but there will be fewer choices at each stage.

* What should we do about the workshop pages and other "hidden" content?

* Any URLs that go away should forward somewhere reasonable or be replaced with a page to manually follow.

Here's the new plan from @andrewgelman:

### Home Page

As is.

### Top-level Menu Items

These will be the menu choices on the home page and the targets of the menu items:

* Interfaces
* Documentation
* Issues
* Community

The first three will remain the same.  

Community will get a landing page with links to all the previous pages:

* Users list and dev list (was what we called "Community" before)
* Bug Reports & Feature Requests 
* Events
* How to Cite & Citations
* Team
* Developer Resources
* Contributions & Supporters

If we can figure it out, it might make sense to have a submenu of some kind.  Otherwise, each of the pages will only have the top-level menu items and it'll be a challenge as to how to lay out that community page index, as it'll require a bit of explanation for each item and some scrolling.

### What we have now

Thsi is just an outline of the current site (as of this issue submission):

Items in CAPS are top-level pages corresponding to the menu items used on all pages.

Each of the entries followed by ++ is its own web page, meaning that some pages just have links out and text, but others have links to other pages on the site.  

HOME++
* what is Stan
* how to get started
* how to donate  (remove NuMFOCUS logo and simplify link to contributions)

INTERFACES++
* Language
  - RStan++, PyStan++, ...
* Higher Level
  - RStanArm++
  - ShinyStan++
* Lower Level
  - Math library++
  - Stan library++
* Useful tools
  - Loo++
  - Bayesplot++
  - Rstantools++

DOCS
* Manual
* Example Models
* Case Studies++
* Tutorials
  - video
  - web tutorials
  - papers
  - books

ISSUES
* Bug Reports
* Feature Requests
* Submit fixes

EVENTS
* meetups
* talks
* courses
* cons

COMMUNITY
* users group
* dev group

CITE
* How to cite Stan
* Books about Stan
* Boosks with examples translated to Stan
* Papers about Stan
* Papers using Stan
* Theses using Stan
* Papers using NUTS
* Software using Stan
* Software using NUTS

TEAM
* Core Developers
* Alum developers

DEVELOPMENT
* suggest or discuss code patches
* contribute code patches
* to do list and projects
* how to join the dev team
* source repos

SHOP
* US shop
* UK shop

SUPPORT
* Contribute to Stan
* Donors
* R&D grants
* Industrial consulting

**Comment by jgabry** [comment]
I've always found the most frustrating part of the website to be the lack of a table of contents (or something along those lines) at the top of each page. That is, not knowing what's on a page without having to scroll through the whole thing. For example, the current Citations page has "How to Cite Stan" as the header, which is fine, but there's a lot more on the page besides what is described by "How to Cite Stan". There are also books about Stan, papers using Stan, books with examples translated into Stan, etc.

Reducing the number of menu items sounds good to me, but then I think it will be even more important to have a table of contents at the top of each page linked to from the menu since there will now be even more content per page (or at least more content linked to from each page).

**Comment by bob-carpenter** [comment]
Good point.  But I don't like tables of contents.  It seems to
be trying to put a band aid on a problem that's more structural.

What if the pages are shorter, so that "how to cite Stan" and
"citations of Stan" are no longer on the same page?    Of course,
it makes the site deeper, but perhaps easier to navigate.  At least
I think that was Andrew's thinking.

- Bob

**Comment by jgabry** [comment]
Yeah I have no particular love for tables of contents, so if we can solve
the problem I described a different way that would be fine from my
perspective. More pages with less content per page is probably better than
having ToCs everywhere. As long as it's possible to know what information
the cite provides (at a high level) without actually having to scroll to
the bottom of a lot long pages it will be an improvement

**Comment by bob-carpenter** [comment]
@jgabry on the mailing list suggested that we not have deep nesting on a page like on the current citations and docs pages (where you have to scroll down to see what's there).  One solution would be a table of contents on each page and another would be shorter (but more deeply nested) pages.
